docs/relational-databases/collations/collation-and-unicode-support.md

```diff
@@ -133,7 +133,7 @@ For example, if you are using the SQL collation **SQL_Latin1_General_CP1_CI_AS**
 
 For more information, see [SQL Server Collation Name (Transact-SQL)](../../t-sql/statements/sql-server-collation-name-transact-sql.md).
 
-During [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] setup, the default installation collation setting is determined by the operating system (OS) locale. You can change the server-level collation either during setup or by changing the OS locale before installation. For backward compatibility reasons, the default collation is set to the oldest available version that's associated with each specific locale. Therefore, this isn't always the recommended collation. To take full advantage of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] features, change the default installation settings to use Windows collations. For example, for the OS locale "English (United States)" (code page 1252), the default collation during setup is **SQL_Latin1_General_CP1_CI_AS**, and it can be changed to its closest Windows collation counterpart, **Latin1_General_100_CI_AS_SC**.
+During [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] setup, the default installation collation setting is determined by the operating system (OS) locale. You can change the server-level collation either during setup or by changing the OS locale before installation. For backward compatibility reasons, the default collation is set to the oldest available version that's associated with each specific locale. Therefore, this is not always the recommended collation. To take full advantage of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] features, change the default installation settings to use Windows collations. For example, for the OS locale "English (United States)" (code page 1252), the default collation during setup is **SQL_Latin1_General_CP1_CI_AS**, and it can be changed to its closest Windows collation counterpart, **Latin1_General_100_CI_AS_SC**.
```

Comment:
```suggestion
During [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] setup, the default installation collation setting is determined by the operating system (OS) locale. You can change the server-level collation either during setup or by changing the OS locale before installation. For backward compatibility reasons, the default collation is set to the oldest available version that's associated with each specific locale. Therefore, this isn't always the recommended collation. To take full advantage of [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] features, change the default installation settings to use Windows collations. For example, for the OS locale "English (United States)" (code page 1252), the default collation during setup is **SQL_Latin1_General_CP1_CI_AS**, and it can be changed to its closest Windows collation counterpart, **Latin1_General_100_CI_AS_SC**.
```